Expensify/App
# Add autoCapitalize prop to avoid first letter duplication on android

### Details

Add `autoCapitalize` prop for `TextInputWithPrefix` and `TextInputWithLabel` (drill)
Pass `autoCapitalize="none"` prop to `TextInputWithLabel` in WorkspaceNewRoomPage

### Fixed Issues

$ https://github.com/Expensify/App/issues/6725

### Tests | QA Steps

1. Log in with an Expensify account
2. Click on global create
3. Create new chat room
4. Start typing on room name

Expected result: You should not get the first letter duplicated

### Tested On


### Screenshots


#### Web


https://user-images.githubusercontent.com/47149004/147614962-18d724d8-2b69-47bb-b794-715f7185b893.mov


#### Mobile Web


#### Desktop


#### iOS


https://user-images.githubusercontent.com/47149004/147614932-1d3d18af-5e60-45ff-be4b-6842a5198a1e.mov



#### Android
https://user-images.githubusercontent.com/47149004/147581857-d00dadea-b2e9-44dc-8918-995f6d1ade0d.mp4
